lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_file_by_version.rb

```diff
@@ -57,6 +58,7 @@ def self.available_options
                                        description: 'The prefix which is used in the GitHub release title',
                                        type: String,
                                        optional: true),
+          Fastlane::Helper::GithubHelper.github_token_config_item
```

Comment:
I just triggered CI on your PR (FYI, because you're working on a fork, CI isn't run automatically—for security reasons— like it is for usual PRs), and got a rubocop warning on this line (see [here](https://buildkite.com/automattic/release-toolkit/builds/778#01841a38-ee51-4d09-856b-2ead2defb277)):

```
lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_file_by_version.rb:61:11: C: [Correctable] Style/TrailingCommaInArrayLiteral: Put a comma after the last item of a multiline array.
          Fastlane::Helper::GithubHelper.github_token_config_item
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
I do not have access to see that buildkite link 😢 
I fixed the warning, hope everything pass now 👍

Reply:
I've retriggered the CI on your latest commit, let's see if CI checks go green on your PR after that 🙂